src/reference/asciidoc/aggregator.adoc

```diff
@@ -187,53 +168,55 @@ public class MyReleaseStrategy {
-`Collection<?>` avoids the conversion but still requires creating the new `Collection`.
+The least efficient is a POJO method with a `Collection<Something>` type. The framework has to copy the payloads from the messages in the group into a new collection (and possibly attempt conversion on the payloads to `Something`) every time the release strategy is called.
+Using `Collection<?>` avoids the conversion but still requires creating the new `Collection`.
 
-*For these reasons, for large groups, it is recommended that you implement `ReleaseStrategy`.*
+For these reasons, for large groups, we recommended that you implement `ReleaseStrategy`.
 =====
 
 When the group is released for aggregation, all its not-yet-released messages are processed and removed from the group.
-If the group is also complete (i.e.
-if all messages from a sequence have arrived or if there is no sequence defined), then the group is marked as complete.
-Any new messages for this group will be sent to the discard channel (if defined).
-Setting `expire-groups-upon-completion` to `true` (default is `false`) removes the entire group and any new messages, with the same correlation id as the removed group, will form a new group.
-Partial sequences can be released by using a `MessageGroupStoreReaper` together with `send-partial-result-on-expiry` being set to `true`.
+If the group is also complete (that is, if all messages from a sequence have arrived or if there is no sequence defined), then the group is marked as complete.
+Any new messages for this group are sent to the discard channel (if defined).
+Setting `expire-groups-upon-completion` to `true` (the default is `false`) removes the entire group, and any new messages (with the same correlation ID as the removed group) form a new group.
+You can release partial sequences by using a `MessageGroupStoreReaper` together with `send-partial-result-on-expiry` being set to `true`.
 
 IMPORTANT: To facilitate discarding of late-arriving messages, the aggregator must maintain state about the group after it has been released.
-This can eventually cause out of memory conditions.
-To avoid such situations, you should consider configuring a `MessageGroupStoreReaper` to remove the group metadata; the expiry parameters should be set to expire groups after it is not expected that late messages will arrive.
-For information about configuring a reaper, see <<reaper>>.
+This can eventually cause out-of-memory conditions.
+To avoid such situations, you should consider configuring a `MessageGroupStoreReaper` to remove the group metadata.
+The expiry parameters should be set to expire groups once a point has been reach after after which late messages are not expected to arrive.
+For information about configuring a reaper, see "`<<reaper>>`".
```

Comment:
Is it really going to work properly?
Our point was to have a link to the appropriate chapter.
Why have wrapped it to the code and to double quotes at the same time?

Reply:
That combination makes curly double quotation marks. Curly double quotation marks are the standard in the publishing industry. Straight quotation marks are used only in code.